src/components/modal/modal.js

```diff
@@ -155,11 +162,13 @@ class Modal extends React.Component {
       // it was added as the Portal library we use messes up the callstack
       // when assigning the ref and triggering componentDidUpdate
       setTimeout(() => {
+        this.updateState();
```

Comment:
`updateState` is a weird name inside of a React component - maybe it should be something more clear?